### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by TruthOFT** [LazyVim/LazyVim#987] [Q&A] [answered]
### Did you check docs and existing issues?

- [X] I have read all the LazyVim docs
- [X] I have searched the existing issues of LazyVim
- [X] I have searched the existing issues of plugins related to this issue

### Neovim version (nvim -v)

0.91

### Operating system/version

Ubuntu 22.04

### Describe the bug

I have installed one of nerd font v3  But it still can't show the icons

### Steps To Reproduce

None

### Expected Behavior

None

### Repro

```Lua
-- DO NOT change the paths and don't remove the colorscheme
local root = vim.fn.fnamemodify("./.repro", ":p")

-- set stdpaths to use .repro
for _, name in ipairs({ "config", "data", "state", "cache" }) do
  vim.env[("XDG_%s_HOME"):format(name:upper())] = root .. "/" .. name
end

-- bootstrap lazy
local lazypath = root .. "/plugins/lazy.nvim"
if not vim.loop.fs_stat(lazypath) then
  vim.fn.system({ "git", "clone", "--filter=blob:none", "https://github.com/folke/lazy.nvim.git", lazypath, })
end
vim.opt.runtimepath:prepend(lazypath)

-- install plugins
local plugins = {
  "folke/tokyonight.nvim",
  "folke/LazyVim",
  -- add any other plugins here
}
require("lazy").setup(plugins, {
  root = root .. "/plugins",
})

vim.cmd.colorscheme("tokyonight")
-- add anything else here
```

**Comment by folke** [comment, marked as answer]
Please use github discussions for questions.

If icons dont work, then your terminal is probably still not using the font or you didnt properly set it up.

Also, make sure you're using a modern terminal.

Either way, this is not a LazyVim issue. I'm sure you'll figure it out

**Reply by TruthOFT** [reply]
OK, Thanks, I just think every 'issue' is an issue

**Reply by folke** [reply]
Not every "issue" is due to the plugin, like in this case this is  100% not caused by LazyVim.
I'm not going to debug everyone's installation. LazyVim requires a properly configured terminal and some other things.

**Reply by TruthOFT** [reply]
got it. Then I think it's caused by the terminal of Ubuntu. May be it's old

**Reply by folke** [reply]
use kitty, alacritty or wezterm

**Reply by TruthOFT** [reply]
Alright, I'm gonna do that,Thanks

**Reply by robsonpeixoto** [reply]
@Eobard-Thawne https://www.youtube.com/watch?v=mQdB_kHyZn8
